<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefDialog</class>
 <widget class="QDialog" name="PrefDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/configure.png</normaloff>:/img/configure.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="mTabsWg">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="mAppearanceTab">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <widget class="QGroupBox" name="mControlsGb">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>451</width>
         <height>118</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="title">
        <string>Controls</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="mMenuBarCb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Show or hide the Menubar.</string>
          </property>
          <property name="text">
           <string>Show Menubar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mToolBarCb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Show or hide the Toolbar.</string>
          </property>
          <property name="text">
           <string>Show Toolbar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mStatusBarCb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Show or hide the Statusbar.</string>
          </property>
          <property name="text">
           <string>Show Statusbar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="mIndicatorsGb">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>451</width>
         <height>171</height>
        </rect>
       </property>
       <property name="title">
        <string>Indicators</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="mShowIndicatorsCb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>If this option is checked, an indicator
inside each peg is displayed.</string>
          </property>
          <property name="text">
           <string>Show Indicators inside pegs</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>30</number>
          </property>
          <item>
           <widget class="QRadioButton" name="mLettersRb">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Use unique letters as indicators.</string>
            </property>
            <property name="text">
             <string>Letter Indicators</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="mNumbersRb">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Use unique numbers as indicators.</string>
            </property>
            <property name="text">
             <string>Number Indicators</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mHideColorsCb">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Use the same neutral background color for each peg,
when indicators are shown.</string>
            </property>
            <property name="text">
             <string>Hide Colors</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="mResetColorOrderBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Click here if you have changed the color order
and now you want to revert it to it's initial state. </string>
          </property>
          <property name="text">
           <string>Reset Color Order</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="mGameTab">
      <attribute name="title">
       <string>Game Settings</string>
      </attribute>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>13</y>
         <width>461</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="mLevelPresetsLbl">
          <property name="text">
           <string>Level Presets</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QComboBox" name="mLevelPresetsCmb">
          <property name="toolTip">
           <string>Select from predefined combinations
of the three different level related settings,
the number of colors, the number of slots (columns)
and the possibility of pegs of the same color in one row. </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="mSlotCntLbl">
          <property name="text">
           <string>Number of Slots</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="mColorCntLbl">
          <property name="text">
           <string>Number of Colors</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QComboBox" name="mPegCntCmb">
          <property name="toolTip">
           <string>Number of pegs in a single row.
Higher values are harder to guess.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="mColorCntCmb">
          <property name="toolTip">
           <string>Number of all possible colors.
Higher values are harder to guess.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QCheckBox" name="mSameColorsCb">
          <property name="toolTip">
           <string>If this option is checked, the hidden combination
may contain one or more (even all would be possible)
pegs of the same color.
If checked, the combination is generally harder to guess.</string>
          </property>
          <property name="text">
           <string>Allow Pegs of Same Color</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="4">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLabel" name="mGameModeLbl">
          <property name="text">
           <string>Game Mode</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QComboBox" name="mGameModeCmb">
          <property name="toolTip">
           <string>Select from the different game modes.
In the 1. mode you have to to guess the hidden,
computer generated combination,
In the 2. mode your computer will try to guess your
own secret combination.</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="4">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="3">
         <widget class="QCheckBox" name="mAutoCloseCb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>If checked, a row is closed as soon as you place the last peg
and your guess is submitted immediatly.
This option only affects the Human vs. Computer game mode.</string>
          </property>
          <property name="text">
           <string>Close Rows when the last Peg is placed</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="2">
         <widget class="QCheckBox" name="mAutoHintsCb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>If checked, the computer will determine the corresponding hints for the solver's guess.
This option affects only Computer vs. Human games.
If checked, you only have to place your secret combination.
Then the rest of the game runs automatically until the end.
But you can set a delay between the guesses.</string>
          </property>
          <property name="text">
           <string>Set Hints automatically</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QSpinBox" name="mDelaySb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Adjust the delay (in ms) between two guesses.</string>
          </property>
         </widget>
        </item>
        <item row="9" column="3">
         <widget class="QLabel" name="mDelayLbl">
          <property name="text">
           <string>Delay in ms</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="4">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="mStrenghtLbl">
          <property name="text">
           <string>Solver Strength</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QComboBox" name="mStrengthCmb">
          <property name="toolTip">
           <string>Select from 3 different strength levels for your computer opponent.
This option only affects the Computer vs. Human game mode.
For the computer's guess in the Human vs. Computer mode
always the highest level will be used.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
